requirements.txt

```diff
@@ -1,12 +1 @@
-atomicwrites==1.4.0
```

Comment:
(minor) This needs double checking on bitbucket - transitive dependencies were not being correctly collected at last check